scripts/start-rc.bat

```diff
@@ -1,7 +1,9 @@
 @echo off
 setlocal EnableDelayedExpansion
 
-set HZ_VERSION=4.1.4-SNAPSHOT
+if "%HZ_VERSION%"=="" (
+    set HZ_VERSION=5.2-SNAPSHOT
```

Comment:
Should we use 5.2 with the 4.1.z branch? I have backported the certificate PR to maintenance Hazelcast branches as well. So, `4.1.10-SNAPSHOT` would also work for this branch.

Reply:
It can be both ways. We should be working in all cases anyway and the during the release compatibility tests will handle it.